clang/test/Driver/clang-offload-bundler-bc-archive-support-linux.cpp

```diff
@@ -10,104 +11,104 @@
 // In all these tests also ensure functionality with bundled object files still
 // works correctly.
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// Make bundled object with targets:
-// sycl-spir64-unknown-unknown
-// host-x86_64-unknown-linux-gnu
-// RUN: %clangxx -fsycl -c %s -o %t_bundled.o
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// Make three distinct BC files
-// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE1 %s -o %t1.bc
-// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE2 %s -o %t2.bc
-// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE3 %s -o %t3.bc
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// Bundle BC files to different targets:
-// host-spir64-unknown-unknown
-// host-spir64_gen
-// host-spir64_x86_64
-// RUN: clang-offload-bundler -type=bc -targets=host-spir64-unknown-unknown -input=%t1.bc  -output=%t1_bundled.bc
-// RUN: clang-offload-bundler -type=bc -targets=host-spir64_gen             -input=%t2.bc  -output=%t2_bundled.bc
-// RUN: clang-offload-bundler -type=bc -targets=host-spir64_x86_64          -input=%t3.bc  -output=%t3_bundled.bc
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// Make archive with bundled BC and o files
-// RUN: rm -f %t_bundled.a
-// RUN: llvm-ar cr %t_bundled.a %t1_bundled.bc %t2_bundled.bc %t3_bundled.bc %t_bundled.o
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Make bundled object with targets:
+// sycl-spir64-unknown-unknown
+// host-x86_64-unknown-linux-gnu
+// RUN: %clangxx -fsycl --no-offload-new-driver -c %s -o %t_bundled.o
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Make three distinct BC files
+// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE1 %s -o %t1.bc
```

Comment:
Is `--no-offload-new-driver` not required here?
I see it added [here](https://github.com/intel/llvm/pull/14539/files?diff=unified&w=0#diff-2691373ddc872311fdaadae7dbfe0eeeaaab35a1fa7b3aa3a145e6d49017535aR22) for generating BC files.

Reply:
It is required - will add.